dataikuapi/dss/dataset.py

```diff
@@ -523,6 +524,16 @@ def get_metric_history(self, metric, partition=''):
                 "GET", "/projects/%s/datasets/%s/metrics/history/%s" % (self.project_key, self.dataset_name, 'NP' if len(partition) == 0 else partition),
                 params={'metricLookup' : metric if isinstance(metric, str) or isinstance(metric, unicode) else json.dumps(metric)})
 
+    def get_info(self):
+        """
+        Retrieve all the information about a dataset
+
+        :returns: a :class:`DSSDatasetInfo` containing all the information about a dataset, such as params, schema, last build infos, status, etc.
```

Comment:
I would probably not mention that it contains "params, schema, status", since we provide no accessors for these.

Beware, "information" is always singular.

Reply:
removed 👍🏻